examples/logging.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"flag"
+	"net/http"
+
+	"../nodego"
+	"log"
+)
+
+func init() {
+	nodego.OverrideLogger()
```

Comment:
Almost everything in GCF is request based, including logging.  GCF inserts middleware in the request path to report logs on a per-request basis, so I don't see why we shouldn't do the same. GCF guarantees that your instance will only receive a single concurrent request to make this possible.

Ex:
```
func LoggingMiddleware(h http.HandlerFunc ) http.HandlerFunc {
	return func(w http.ResponseWriter, r *http.Request) {
		startLogging()
		h(w, r)
		endLogging()
	}
}

func main() {
	flag.Parse()
	http.HandleFunc(nodego.HTTPTrigger, LoggingMiddleware(func(w http.ResponseWriter, r *http.Request) {
		log.Println("This is a log message from Go!")
	}))

	nodego.TakeOver()
}
```

Reply:
How should the cleanup work? It doesn't look like it's possible to get the existing writer from the std logger.